examples/assets_smoke_test/dbt_project/config/profiles.yml

```diff
@@ -0,0 +1,28 @@
+dbt_project:
+  target: local
+
+  outputs:
+    local:
+      type: snowflake
+      account: "{{ env_var('SNOWFLAKE_ACCOUNT') }}"
+
+      user: "{{ env_var('SNOWFLAKE_USER') }}"
+      password: "{{ env_var('SNOWFLAKE_PASSWORD') }}"
+
+      database: "{{ env_var('SNOWFLAKE_DATABASE') }}"
+      schema: PUBLIC
+      warehouse: ELEMENTL
+      client_session_keep_alive: False
+
+
+    smoke_test:
```

Comment:
why have two targets here if their config is identical? should we potentially write the smoke test outputs to a different schema or database?

Reply:
the thinking was that in real life they'd end up varying. I'll vary the names of the env variables to make that more clear